### ISSUE TITLE
Follow up with sites that audit determined look close to launching

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#8063]
# Story

As a web team, we would like to follow up with SiteNow v1 Standard sites that appear close to launch ready to see what is needed to get them across the finish line.

# Additional Info

In https://github.com/uiowa/uiowa/issues/7886, we added some additional info to the spreadsheet to capture some idea of progress. We should follow up with the ones that appear launch ready to see if we can get them to done.

Some of them had extensions we granted and those extension dates have now passed or will soon.

Should be able to filter the spreadsheet by standard, active, requested a v3 site. Should be ~8 I believe.

```[tasklist]
### Responded
- [x] https://studiovictorianfboos.prod.drupal.uiowa.edu/ - requested two week extension until 10/10/24 - emailed on 10/10/24 - launched and v1 removed
- [x] https://labanthony.prod.drupal.uiowa.edu - launched
- [x] https://labhultman.prod.drupal.uiowa.edu - launched
- [x] https://labricherson.prod.drupal.uiowa.edu/ - launched
- [x] https://labcampbell.prod.drupal.uiowa.edu - launched
- [x] https://sitesuisrmsymposium.prod.drupal.uiowa.edu/ - launched
- [ ] https://lawnhlp.prod.drupal.uiowa.edu/ - Essentially, they’re planning to finish the migration, but Laura needs to check in with everyone first to make sure the v3 site is good to go. - sent a follow up email on 10/09/24
```

```[tasklist]
### No response
- [ ] https://lablira.prod.drupal.uiowa.edu/ - Sent a follow up email on 10/09/24
```

**Comment by briand44** [comment]
Grooming conversation... Discussed not including unit leaders in these communications unless we need to because we are not getting responses, etc. Send from its-web for visibility.

**Comment by joewhitsitt** [comment]
I am going to start drafting something in ITS-web and getting a list of emails and sites to populate with.

**Comment by joewhitsitt** [comment]
Subject: Request to proceed with launching your replacement SiteNow website

Hi,

We hope this message finds you well. As you know, we are in the final stages of retiring our Drupal 7 based SiteNow v1 Standard platform. We appreciate your cooperation throughout this transition process.

We wanted to remind you of the original August 1st deadline for the transition. After reviewing, it looks like your new SiteNow website is potentially ready for launch. We understand you requested an extension, which has now passed. _// This is an optional sentence that can be removed if the website didn't have an extension._

We wanted to confirm with you before proceeding with the launch process. Please let us know if we can go ahead and launch your replacement SiteNow v3 website at your current URL. Once the new website is live, we will proceed with removing the old SiteNow v1 website.

Current URL (v1 Standard): [Current URL]
Staging URL: [Staging URL]

We look forward to your confirmation. If you have any concerns or need further assistance, please don’t hesitate to reach out. We are here to ensure a smooth transition for you.

Thank you,

Joe and the OSC Web Team

**Comment by joewhitsitt** [comment]
Looks like anthony.lab (7/8) and hultman.lab (8/27) have pre-launch threads where they are needing to set a site email and/or delete/archive/repurpose demo content.

updated some other contacts based on its-web emails and users on the new site.

**Comment by joewhitsitt** [comment]
I have sent the emails. Mentioning some pre-launch items for certain ones.

**Comment by briand44** [comment]
I'm not ready to close this yet. I would like to review responses and continue pushing on these until they have launched.

**Comment by briand44** [comment]
Going to close this as the remaining sites here will be included in #8018
